staging/src/k8s.io/api/admissionregistration/v1/types.go

```diff
@@ -186,7 +186,7 @@ type ValidatingWebhook struct {
 	// Rules describes what operations on what resources/subresources the webhook cares about.
 	// The webhook cares about an operation if it matches _any_ Rule.
 	// However, in order to prevent ValidatingAdmissionWebhooks and MutatingAdmissionWebhooks
-	// from putting the cluster in a state which cannot be recovered from without completely
```

Comment:
I'm not the expert but the "from" seems correct to me.

Reply:
Oops, my mistake, i'll restore it